db/migrate/20200405233913_add_no_sp_tracking_to_doc_auth_logs.rb

```diff
@@ -0,0 +1,8 @@
+class AddNoSpTrackingToDocAuthLogs < ActiveRecord::Migration[5.1]
+  def change
+    safety_assured do
```

Comment:
Why do we need `safety_assured` here? If we do need it we should probably throw in a comment explaining what it is okay to use `safety_assured` here.